packages/apps/plugins/plugin-debug/src/DebugPlugin.tsx

```diff
@@ -215,12 +215,11 @@ export const DebugPlugin = (): PluginDefinition<DebugPluginProvides> => {
             case 'main':
               return active === 'devtools' ? (
                 <DevtoolsMain />
-              ) : !active || typeof active !== 'object' ? null : 'space' in active &&
-                active.space instanceof SpaceProxy ? (
+              ) : !active || typeof active !== 'object' ? null : 'space' in active && isSpace(active.space) ? (
                 <DebugSpace
                   space={active.space}
                   onAddObjects={(objects) => {
-                    if (!(active.space instanceof SpaceProxy)) {
+                    if (!isSpace(active.space)) {
```

Comment:
do we need this? `instance of Space` would be fine?

Reply:
`Space` is not a class it's an interface so that won't work

Reply:
If we can find a way to avoid needing this helper that'd be great though

Reply:
Maybe if it was an abstract class? Not sure if that works with instanceof or not